### PR TITLE
docs(map_projection_loader): add projector type image

### PR DESCRIPTION
**Pull request by KYabuuchi**
## Description

I visualized how each projector type projects coordinates.

#### Reference: LocalCartesianUTM
1. [geography_utils::get_lanelet2_projector()](https://github.com/autowarefoundation/autoware.universe/blob/e7a826858bd4164701b26a2f2d0c059ba80606fc/common/geography_utils/src/lanelet2_projector.cpp#L27-L32)
2.  [UtmProjector::UtmProjector()](https://github.com/fzi-forschungszentrum-informatik/Lanelet2/blob/4789fde6c3bafcd5a8d721d11c5d92870ca59a22/lanelet2_projection/src/UTM.cpp#L8-L13)
3.  [UTMUPS::Forward()](https://github.com/geographiclib/geographiclib/blob/2fa1c84ab172305701127d116921a4f3c64b9a79/src/UTMUPS.cpp#L79-L90)
 
####  Reference: TransverseMercator
1. [geography_utils::get_lanelet2_projector()](https://github.com/autowarefoundation/autoware.universe/blob/e7a826858bd4164701b26a2f2d0c059ba80606fc/common/geography_utils/src/lanelet2_projector.cpp#L40-L47)
2. [TransverseMercatorProjector::TransverseMercatorProjector()](https://github.com/autowarefoundation/autoware_common/blob/b92f64f582f498dee40ecc6a6761c8b79cda37c0/tmp/lanelet2_extension/lib/transverse_mercator_projector.cpp#L29-L34)

## Tests performed

This PR changes only documentation.

## Effects on system behavior

This PR changes only documentation.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: map/map_projection_loader/docs/map_projector_type.svg
Comment (kminoda) [inline map/map_projection_loader/docs/map_projector_type.svg:1]: `father` :man: -> `farther`

Comment (kminoda) [inline map/map_projection_loader/docs/map_projector_type.svg:1]: `Outside the mgrs ...` -> `Outside of the mgrs ...`

Comment (kminoda) [inline map/map_projection_loader/docs/map_projector_type.svg:1]: How about 
`larger projection error father from the meridian` -> `farther away from the meridian, the greater the projection error becomes`?

**Review by kminoda — COMMENTED** [review]
Thank you very much :rocket: 
The figure LGTM in terms of technical accuracy. I left some nit comments on the grammar and explanation.

**Comment by KYabuuchi** [comment]
@kminoda Thank you for your review! :+1:  I have reflected all suggestions in [67c8580](https://github.com/autowarefoundation/autoware.universe/pull/7062/commits/67c8580d16e94bd58a44cb79e998325a7e64d11f)

File: map/map_projection_loader/docs/map_projector_type.svg
Comment (YamatoAndo) [inline map/map_projection_loader/docs/map_projector_type.svg:1]: `greather` -> `greater`

Comment (KYabuuchi) [reply map/map_projection_loader/docs/map_projector_type.svg:1]: Sorry...  fixed in 2923209

**Review by kminoda — APPROVED** [review]
LGTM
Nice drawing :100:

**Pull request merged**